config.yml

```diff
@@ -3,10 +3,16 @@ accounts:
     coins: ["10000000mand", "200000000cred"]
   - name: bob
     coins: ["10000000mand", "100000000cred"]
-  - name: potato
-    coins: ["3000000mand", "10000000cred"]
   - name: carrot
     coins: ["9000000mand", "30000000cred"]
+  - name: frooti
+    coins: ["7000000mand", "50000000cred"]
+  - name: lemon
+    coins: ["9000000mand", "50000000cred"]
+  - name: melon
+    coins: ["8000000mand", "50000000cred"]
+  - name: potato
+    coins: ["3000000mand", "10000000cred"]
```

Comment:
apart from potato, we don't need cred on any other accounts as well. let's remove them as well